packages/ruleset/test/merge-patch-optional-properties.test.js

```diff
@@ -2,12 +2,12 @@ const { mergePatchOptionalProperties } = require('../src/rules');
 const { makeCopy, rootDocument, testRule, severityCodes } = require('./utils');
 
 const rule = mergePatchOptionalProperties;
-const ruleId = 'ibm-merge-patch-optional-properties';
+const ruleId = 'ibm-merge-patch-properties';
```

Comment:
I'm just wondering, why are the rule name and id different here? (`mergePatchOptionalProperties` vs `ibm-merge-patch-properties`)
I see there are a few cases where they are not the same, so I guess this is intentional.

Reply:
done...